data_reports/111_USGSMuhs.Rmd

```diff
@@ -0,0 +1,199 @@
+USGS Muhs
+
+In this section, we will pull out `USGS Muhs` from ISCN3 and clean it up. 
+Empty columns were removed from the dataset and missing values were filled in. This data set required a cryptic message in the profile of the dataset to be formally cited using BibTeX. (https://doi.org/10.1016/S0277-3791(03)00167-7)
```

Comment:
Need a few more details here. Were there any ISCN gap filled values, if so how were those dealt with? Were there any duplicate data in the rows?

Reply:
Also need citation using the @ format up here instead of giving the doi.

Reply:
Fixed